### DISCUSSION TITLE
配置模块的问题

### DISCUSSION THREAD

**Discussion by rose8928** [opensumi/core#2136] [Q&A] [answered]
想问一下配置模块可以自己写组件吗，有相关例子吗？我自己测试，react不太熟悉，代码和报错如下图：（Test是我随便写的一个简单组件）
![12 28-0](https://user-images.githubusercontent.com/35814595/209791789-13430321-0cfc-4959-8e09-e3f7c46e4b20.png)
![12 28](https://user-images.githubusercontent.com/35814595/209791807-2ed279bf-7dfb-47f3-91e2-cc21396b95af.png)

**Comment by erha19** [comment, marked as answer]
@rose8928 配置面板承载的内容就是配置，目前是不支持自定义内容的，你可以通过注册配置的方式拓展面板上的内容，见 https://opensumi.com/zh/docs/develop/basic-design/contribution-point#%E9%85%8D%E7%BD%AE%E6%B3%A8%E5%86%8C

**Comment by rose8928** [comment]
了解了，谢谢
